pom.xml

```diff
@@ -22,7 +22,7 @@
     <parent>
         <groupId>org.apache.sling</groupId>
         <artifactId>sling-bundle-parent</artifactId>
-        <version>41</version>
+        <version>40</version>
```

Comment:
I am gonna upgrade to the newest parent post merge so we can ignore this downgrade